ansible/configs/ocp4-cluster/files/requirements_k8s.txt

```diff
@@ -1,5 +1,5 @@
 adal==1.2.2
-ansible==2.9.4
+ansible==2.9.6
```

Comment:
Just making sure, you have moved all the previously named `k8s` and `os` modules from their 2.8 names to 2.9 names?

Reply:
This is the latest pip freeze list.